src/Builder/ArchiveBuilderHelper.php

```diff
@@ -74,6 +79,10 @@ public function isSkippable(PackageInterface $package): bool
         return false;
     }
 
+    /**
+     * @param array<int, string> $names
+     * @param array<int, string> $list
```

Comment:
For lists, the syntax is `list<Type>` ([doc](https://phpstan.org/writing-php-code/phpdoc-types#lists))

```suggestion
     * @param list<string> $names
     * @param list<string> $list
```

Reply:
Awesome, didn't realize that was a thing. I'll update my PR to update all instances of `array<int, T>` with `list<T>`